d365fo.tools/functions/get-d365webservertype.ps1

```diff
@@ -0,0 +1,38 @@
+
+<#
+    .SYNOPSIS
+        Get the default web server to be used
+        
+    .DESCRIPTION
+        Get the web server which will be used to run D365FO: Either IIS or IIS Express.
+        Newly deployed development machines will have this set to IIS Express by default.
+
+    .EXAMPLE
+        PS C:\> Get-D365WebServerType
+        
+        This will display the current web server type registered in the "DynamicsDevConfig.xml" file.
+        Located in "K:\AosService\PackagesLocalDirectory\bin".
+        
+    .NOTES
+        Tag: Web Server, IIS, IIS Express, Development
+        
+        Author: Sander Holvoet (@smholvoet)
+   
+#>
+
+function Get-D365WebServerType {
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseSingularNouns", "")]
+    [CmdletBinding()]
+    param ()
+
+    $filePath = "K:\AosService\PackagesLocalDirectory\bin\DynamicsDevConfig.xml"
```

Comment:
You can use the ```$Script:ServiceDrive``` variable to get the drive of the AosService folder. See [Add-D365WindowsDefenderRules](https://github.com/d365collaborative/d365fo.tools/blob/c34e0fd70311f7b368c618f0a54c29a917428d53/d365fo.tools/functions/add-d365windowsdefenderrules.ps1) for an example.

Reply:
We will be using the `$Script:PackageDirectory` variable, which is more common across the module.